frontend/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
Can you copy this file? Should have everything we want. https://github.com/LucasPickering/rps/blob/master/webapp/.eslintrc.js